validate/validate.py

```diff
@@ -95,7 +95,7 @@ def run_validate(args:Union[ValidateArgs,Dict[str, Union[str,list]]]):
 
     all_files_pass = True
 
-    for path in [Path(pathname) for pathname in args.path]:
+    for path in [Path(pathname).resolve() for pathname in args.path]:
```

Comment:
Consider adding tests for this, as well as symlink tests for the other public methods in this file. Alternatively, because this is a public library, we may want to consider either 1. using `resolve()` in all methods that take a `Path`; or 2. encapsulating the methods in a class and making them private (prefix with `_`); or 3. renaming the methods with the `_` prefix w/out encapsulating them in a class.

Reply:
Updated to add the `_` prefix and added tests for the symlink part of the function